### PR TITLE
First site parameters; repository setup

### PR DESCRIPTION
**Pull request by GernotMaier**
This PR sets the first site parameters (altitude) and adds items to the general repository setup:

- simple parameter file [verified_model/Site/South/altitude.json](verified_model/Site/South/altitude.json) and schema [schema/altitude.schema.yml](schema/altitude.schema.yml)
- pre-commit update
- github action validation workflow to check the schema and parameter files.

### CODE REVIEW

**Comment by GernotMaier** [comment]
@orelgueta - I suggest to review this one now, although only altitude has been added. This is really more an 'infrastructure / repository setup' PR. 

We will have to discuss what to put exactly in the metadata section, for now it consists mostly of placeholders or nulls.

File: .github/dependabot.yml
Comment (orelgueta) [inline .github/dependabot.yml:1]: I assume this is unrelated to this PR? Either way, it looks fine

File: .github/workflows/schema-validation.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+jobs:
+  schema-validation:
+    runs-on: ubuntu-latest
+    container: ghcr.io/gammasim/simtools-prod:latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Install findutils
```
Comment (orelgueta) [inline .github/workflows/schema-validation.yml:24]: If you are running on Ubuntu here, do you really need it?

File: .github/workflows/schema-validation.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        run: |
+          microdnf install -y findutils
+
+      - name: Validate schema files.
+        run: |
+          for file in $(find schema -name "*.yml"); do
+            echo "Testing schema $file"
+            simtools-validate-file-using-schema \
+              --file_name $file
```
Comment (orelgueta) [inline .github/workflows/schema-validation.yml]: Should the log level here also be DEBUG?

File: .pre-commit-config.yaml
Comment (orelgueta) [inline .pre-commit-config.yaml:1]: Again, assume this is not related to this PR

File: schema/altitude.schema.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+%YAML 1.2
+---
+title: Schema for altitude model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/data.metaschema.yml
+meta_schema_version: 0.1.0
+name: altitude
+description: |-
+  Altitude above sea level of central reference point.
```
Comment (orelgueta) [inline schema/altitude.schema.yml:10]: This goes back to what we said the other day. Is this the altitude of the reference point or the observation level of CORSIKA?

File: verified_model/Site/South/altitude.json [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            "REFERENCE": {
+                "VERSION": "1.0.0"
+            },
+            "CONTACT": {
+                "ORGANIZATION": "CTAO",
+                "NAME": "CTAO Simulation Team",
+                "EMAIL": null
+            },
+            "PRODUCT": {
+                "DESCRIPTION": "Altitude above sea level of site centre.",
```
Comment (orelgueta) [inline verified_model/Site/South/altitude.json:17]: I am a bit worried about the duplication here with respect to the schema. Is this the entry by the user?

**Review by orelgueta — COMMENTED** [review]
Generally it all looks fine, but I would like to talk 5 minutes about it in order to clear a few things up.

File: .github/dependabot.yml
Comment (GernotMaier) [reply .github/dependabot.yml:1]: it is part of "repository setup".

File: .github/workflows/schema-validation.yml (hunk 1 above)
Comment (GernotMaier) [reply .github/workflows/schema-validation.yml:24]: We are running in the simtools-prod container, where no findutils are installed.

File: .github/workflows/schema-validation.yml (hunk 2 above)
Comment (GernotMaier) [reply .github/workflows/schema-validation.yml]: Correct, added.

File: .pre-commit-config.yaml
Comment (GernotMaier) [reply .pre-commit-config.yaml:1]: it is part of "repository setup".

File: schema/altitude.schema.yml (hunk 3 above)
Comment (GernotMaier) [reply schema/altitude.schema.yml:10]: It is the altitude of the central reference point and not observation level. It will be renamed to `center_altitude` in the next pull request.

File: verified_model/Site/South/altitude.json (hunk 4 above)
Comment (GernotMaier) [reply verified_model/Site/South/altitude.json:17]: This is correct and it also caught my eye this morning. 

PRODUCT:DESCRIPTION is a required entry in the CTA data model, saying "Human-readable description of data product". Not sure what the best solution is, you are absolutely right with the duplication.

File: .github/dependabot.yml
Comment (orelgueta) [reply .github/dependabot.yml:1]: oh, I didn't even notice it's this new repository...

**Review by orelgueta — APPROVED** [review]
